components/Nav/NavListItem/NavListItem.js

```diff
@@ -29,6 +29,13 @@ export default class NavListItem extends Component {
     areSublinksVisible: false,
   };
 
+  constructor() {
```

Comment:
We're using [@babel/plugin-proposal-class-properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) (hidden by the next preset for babel) which automatically binds methods to the class instance if you use arrow methods!

So ditch the whole constructor, and just do this in the future.

```javascript
areSublinksVisible = (event) => {
}
```